src/entities/InducteePointsView.ts

```diff
@@ -0,0 +1,39 @@
+import { ViewEntity, Connection, ViewColumn } from 'typeorm';
+import { AppUser } from './AppUser';
+import { Attendance } from './Attendance';
+import { Event } from './Event';
+
+// later do some grouping by quarter
+@ViewEntity({
+  expression: (connection: Connection) =>
+    connection
+      .createQueryBuilder()
+      .select('appUser.id', 'user')
+      .addSelect('SUM(attendance.duration)', 'points')
+      .addSelect(
+        "SUM(CASE WHEN event.type = 'professional' THEN 1 ELSE 0 END)::int::bool",
```

Comment:
Is this a type cast to int and then bool?

Reply:
yes - we can clean it up with exists later